gui/WorldView.cxx

```diff
@@ -976,6 +978,59 @@ void WorldView::computeContour(double threshold)
   }
 }
 
+//-----------------------------------------------------------------------------
+void WorldView::loadMesh(QString const& path)
+{
+  QFileInfo check_file(path);
+  if (!check_file.exists() || !check_file.isFile())
+  {
+    return;
+  }
+  // Create the vtk pipeline
+  // Read volume
+  vtkNew<vtkPLYReader> reader;
+  reader->SetFileName(qPrintable(path));
+  reader->Update();
+
+  vtkSmartPointer<vtkPolyData> mesh = reader->GetOutput();
+  this->setMesh(mesh);
+}
+
+//-----------------------------------------------------------------------------
+void WorldView::setMesh(vtkSmartPointer<vtkPolyData> mesh)
+{
+  QTE_D();
+
+  mesh->GetPointData()->SetActiveScalars("RGB");
+
+  // Create mapper
+  vtkNew<vtkPolyDataMapper> meshMapper;
+  meshMapper->SetInputData(mesh);
+  meshMapper->ScalarVisibilityOn();
+  meshMapper->SetScalarModeToUsePointFieldData();
+  meshMapper->SelectColorArray("RGB");
```

Comment:
@danlipsa I'm trying to read a PLY mesh (with color per vertex) back into TeleSculptor so we can reload the meshes we export without needing to recompute them.  I've got this working except for applying the color to the mesh.  Can you help me figure out what I'm doing wrong?  What is the right way to tell VTK to use the colors from the RGB array to color the mesh?

Reply:
FWIW, the first thing I'd probably check is if `"RGB"` is the correct name...

Second would be if there's an issue with the way the `volumeActor` is set up.

Reply:
"RGB" is the name.  The data loads in ParaView and has that name.

This is somewhat of a hack in that I'm hijacking the mapper/actor that is also used to show the mesh after contour extraction from the volume.  There may be a better way to manage mesh vs volume in the future.  Maybe there is something else in the code that disables the color.  Or I could just be doing it totally wrong.